pulsar-metadata/src/main/java/org/apache/pulsar/metadata/impl/AbstractMetadataStore.java

```diff
@@ -75,6 +78,11 @@
     protected abstract CompletableFuture<Boolean> existsFromStore(String path);
 
     protected AbstractMetadataStore() {
```

Comment:
I think you should config `operationTimeoutSeconds` via PulsarService#createConfigurationMetadataStore.
And we can re-use the `zooKeeperOperationTimeoutSeconds` in ServiceConfiguration.

Reply:
I will `metadataStoreOperationTimeoutSeconds` add to ServiceConfiguration, and use `metadataStoreOperationTimeoutSeconds` instead of `zooKeeperOperationTimeoutSeconds` is more clear.